arangod/Aql/QueryOptions.h

```diff
@@ -61,6 +61,7 @@ struct QueryOptions {
   size_t memoryLimit;
   size_t maxNumberOfPlans;
   size_t maxWarningCount;
+  size_t timeout;
```

Comment:
let's make this a double.

Reply:
We should also make clear (using code comments) what the difference between `timeout` and `ttl` (two lines) below is. They will be easily confused otherwise.